transform-array.js

```diff
@@ -0,0 +1,9 @@
+const arrNum = new Array(2, 1, 4, 3, 6, 5, 8, 7, 10, 9);
+
+// function that double the odd number in an array and reverse the position
+const transformArrayOfNumbers = (arrNum) => {
+  const oddNum = arrNum.filter((num) => num % 2 === 1);
+  const doubleNum = oddNum.map((num) => num * 2);
+  const reverseNum = doubleNum.reverse();
```

Comment:
The only one comment I have is that you could have returned from here and saved an extra variable, so something like `return doubleNum.reverse()`